packages/sdk/src/sdk/api/generated/default/models/DecodedUserToken.ts

```diff
@@ -27,6 +27,12 @@ import {
  * @interface DecodedUserToken
  */
 export interface DecodedUserToken {
+    /**
+     * 
+     * @type {string}
+     * @memberof DecodedUserToken
+     */
+    apiKey: string;
```

Comment:
@raymondjacobson this u?

Reply:
ya, i think these don't have to all go in 1-1 if it's not feature blocking 🤷 lmk if u disagree and ill be better about this

Reply:
is it ok to go in? just easier to git add the whole thing if so

Reply:
oh yea ofc

Reply:
oh i see what u sayin, and yea agree! just double checkin